docs/docs/integrations/woodpecker-ci.md

```diff
@@ -0,0 +1,17 @@
+# Woodpecker CI
+
+This is a simple example configuration `.woodpecker/trivy.yml` that shows how you could get started:
+
+```yml
+pipeline:
+  securitycheck:
+    image: aquasec/trivy:latest
+    commands:
+      - trivy fs --exit-code 0 --skip-dirs web/ --skip-dirs docs/ --severity UNKNOWN,LOW .
+      - trivy fs --exit-code 1 --skip-dirs web/ --skip-dirs docs/ --severity MEDIUM,HIGH,CRITICAL .
```

Comment:
Is there a reason that these commands have to be so "comprehensive"? I would like to just have a simple example and then mention below that the command can be replaced with any trivy command. What do you think?

Reply:
well what about delete L10 and add a comment about `# use any trivy command, if exit code is 0 woodpecker marks it as passed, else it assumes it failed` ?